api/build.go

```diff
@@ -411,6 +411,96 @@ func skipEmptyBuild(p *pipeline.Build) string {
 	return ""
 }
 
+// swagger:operation GET /api/v1/search/builds/{id} builds GetBuildByID
+//
+// Get a single build by its id in the configured backend
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - in: path
+//   name: id
+//   description: build id
+//   required: true
+//   type: number
+// security:
+//   - ApiKeyAuth: []
+// responses:
+//   '200':
+//     description: Successfully retrieved build
+//     schema:
+//       "$ref": "#/definitions/Build"
+//   '400':
+//     description: Unable to retrieve the build
+//     schema:
+//       "$ref": "#/definitions/Error"
+//   '500':
+//     description: Unable to retrieve the build
+//     schema:
+//       "$ref": "#/definitions/Error"
+
+// GetBuildByID represents the API handler to capture a
+// build by its id from the configured backend.
+func GetBuildByID(c *gin.Context) {
+	// Variables that will hold the library types of the build and repo
+	var (
+		b *library.Build
+		r *library.Repo
+	)
+
+	// Capture user from middleware
+	u := user.Retrieve(c)
+
```

Comment:
can we add a simple log here like `GetBuild` ?
https://github.com/go-vela/server/blob/c565a1d762bbd835b4f05f3d7318b9eda296757d/api/build.go#L955-L963